src/app/clusters/network-commissioning/network-commissioning.cpp

```diff
@@ -135,6 +135,24 @@ void Instance::Shutdown()
     mpBaseDriver->Shutdown();
 }
 
+#if !CHIP_DEVICE_CONFIG_SUPPORTS_CONCURRENT_CONNECTION
+void Instance::SendNonConcurrentConnectNetworkResponse()
+{
+    auto commandHandleRef = std::move(mAsyncCommandHandle);
+    auto commandHandle    = commandHandleRef.Get();
+    if (commandHandle == nullptr)
+    {
+        return;
+    }
+
+    chip::DeviceLayer::ConnectivityMgr().GetBleLayer()->mTerminateOnPacketTxComplete = true;
```

Comment:
```suggestion
    DeviceLayer::ConnectivityMgr().GetBleLayer()->mTerminateOnPacketTxComplete = true;
```
This is all inside `namespace chip`.

Reply:
Done.